test/clj_sqs_extended/core_test.clj

```diff
@@ -8,52 +8,67 @@
 
 (use-fixtures :once fixtures/with-localstack-environment)
 
-(def ^:private test-message-1
-  {:quote "State. You're doing it wrong."
-   :by "Rich Hickey"})
-
-(def ^:private test-message-2
-  {:quote "Nothing says 'Screw You!' like a DSL."
-   :by "Stuart Halloway"})
+(def ^:private test-messages
+  [{:quote "State. You're doing it wrong."
+    :by    "Rich Hickey"}
+   {:quote "Nothing says 'Screw You!' like a DSL."
+    :by    "Stuart Halloway"}
+   {:quote "I don't care if it works on your machine! We are not shipping your machine!"
+    :by    "Vidiu Platon"}
+   {:quote "If you can't beat your computer at chess, try kickboxing."
+    :by    "Claudio E. Montero"}
+   {:quote "I would rather be without a state than without a voice."
+    :by    "Edward Snowden"}])
```

Comment:
it's fun with one or two of these. but this is now taking up 10 lines of code for no reason. can we just generate some random messages instead?